build.sh

```diff
@@ -83,3 +84,17 @@ if [ "$1" != "clean" ]; then
 	copy build/floppy
 	copy resources
 fi
+
+# Create hdd
+if [ "$1" != "clean" ]; then
+	mkdir -p "$WORKSPACE_DIR/build"
+	mkdir -p "$WORKSPACE_DIR/build/hdd"
+	HDD_IMG="$WORKSPACE_DIR/build/hdd.img"
+	# Remove old floppy img
+	rm -f "$HDD_IMG"
+
+	# Make and format the floppy
+	/sbin/mkfs.msdos -C "$HDD_IMG" 1440
```

Comment:
move hdd size to a variable
```suggestion
	/sbin/mkfs.msdos -C "$HDD_IMG" $HDD_SIZE
```